Subject: [PATCH] Fix build if no compression is selected.
---
 configure.ac    | 2 +-
 create-links.pl | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 6110846..a5a488b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -93,7 +93,7 @@ AS_IF(
   [
     AC_MSG_RESULT([yes])
     AC_SUBST([compressor], [none])
-    AC_SUBST([comp_extension], [])
+    AC_SUBST([comp_extension], ["''"])
   ],
   [
     AC_MSG_RESULT([no])
diff --git a/create-links.pl b/create-links.pl
index b2af92e..063128c 100644
--- a/create-links.pl
+++ b/create-links.pl
@@ -21,8 +21,8 @@ use warnings;
 # Get the installation path and compression extension from Makefile
 my $install_path = $ARGV[0]
   or die "Please specify the installation path.";
-my $comp_extension = $ARGV[1]
-  or die "Please specify the compression extension.";
+my $comp_extension = $ARGV[1];
+#  or die "Please specify the compression extension.";
 my $filename = $ARGV[2]
   or die "Please specify the filename with links.";
 
-- 
2.23.0

